src/components/SquaredIconButton/SquaredIconButton.tsx

```diff
@@ -23,8 +23,8 @@ const SquaredIconButton: React.FC<SquaredIconButtonProps> = ({
           height: '20px'
         }
       : {
-          width: '24px',
-          height: '24px'
+          width: '20px',
+          height: '20px'

```

Comment:
I would check this out

Reply:
It was done in that way because the mobile view has another design (whiter border as an example) and we needed it to be 20x20 instead of 24x24.